### DISCUSSION TITLE
Request to add Support for Magic Eden Wallet

### DISCUSSION THREAD

**Discussion by Tcadle434** [rainbow-me/rainbowkit#1988] [Ideas]
Hello! I work for Magic Eden on the Wallet team and wanted to start a convo to see what steps we can take to get the ME wallet integrated with Rainbowkit. We've had several requests for it.

I was looking through the "custom wallet" option in your docs here:
https://www.rainbowkit.com/docs/custom-wallets

But regardless - I think it would be beneficial if we could get integrated with the official wallets from your `@rainbow-me/rainbowkit/wallets` package so teams can easily add us in to their wallet lists without too much extra code

Happy to chat over telegram if it is easier

**Comment by 0xmkh** [comment]
@Tcadle434 We would love to add Magic Eden Wallet to RainbowKit. May i ask you if the wallet supports any EVM chains ?

**Reply by Tcadle434** [reply]
Yes! We support ETH L1, Base and Polygon

**Comment by DanielSinclair** [comment]
@Tcadle434 The easiest way would be to raise a PR and we'll have a look! It's a fairly simple addition, and you can reference an example for Kraken Wallet here: https://github.com/rainbow-me/rainbowkit/tree/main/packages/rainbowkit/src/wallets/walletConnectors/krakenWallet

**Reply by Tcadle434** [reply]
thanks! Opened a PR here: https://github.com/rainbow-me/rainbowkit/pull/1998
